modules/nw-ingress-creating-a-route-via-an-ingress.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+[id="nw-ingress-creating-a-route-via-an-ingress_{context}"]
+= Creating a Route via an Ingress
+
+Some ecosystem components have an integration with Ingress resources but not with
+Route resources. To cover this case, {product-title} automatically creates
+managed Route objects when an Ingress object is created. These Route objects are deleted
+when the corresponding Ingress objects are deleted.
+
+.Procedure
+
+Define an Ingress object:
```

Comment:
Could we add something along the lines of:

. Define an Ingress object in the {product-title} console or by entering the `oc create` command: